scripts/linters/ts_ignore_exceptions.json

```diff
@@ -0,0 +1,145 @@
+{
```

Comment:
Should we also add line numbers here? So that people do not add the same errors but on different lines?

Reply:
I don't think that would work because people might have to change the line numbers here again and again when they edit these files.